repose-aggregator/components/filters/valkyrie-authorization/src/main/resources/META-INF/schema/config/valkyrie-authorization.xsd

```diff
@@ -186,11 +199,11 @@
             <!-- We'll add this when we come back to support xml. At that point the json should switch to a minimum of 0
                  and validation should be in place so that they have at least one of the two specified.-->
             <!--<xs:element name="xml" type="pathTriplet" minOccurs="0" maxOccurs="1">-->
-                <!--<xs:annotation>-->
-                    <!--<xs:documentation>-->
-                        <!--<html:p>Specifies the x-paths necessary to make changes to the collection.</html:p>-->
-                    <!--</xs:documentation>-->
-                <!--</xs:annotation>-->
+            <!--<xs:annotation>-->
+            <!--<xs:documentation>-->
+            <!--<html:p>Specifies the x-paths necessary to make changes to the collection.</html:p>-->
+            <!--</xs:documentation>-->
+            <!--</xs:annotation>-->
```

Comment:
It doesn't bother me enough to want it changed, but just enough b!+&cent;# about it. ;-)